src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -155,8 +162,13 @@ public function convert(FileVersion $fileVersion, $formatKey, $imageFormat)
     {
         $imageResource = $this->mediaImageExtractor->extract($this->storage->load($fileVersion->getStorageOptions()));
 
+        $imagine = $this->imagine;
+        if ('svg' === $imageFormat && class_exists(SvgImagine::class)) {
+            $imagine = new SvgImagine();
```

Comment:
Wouldn't it be better for performance and testability if we inject that as a service?